.github/workflows/pr-check.yml

```diff
@@ -0,0 +1,38 @@
+name: jbt-pr-check
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }} # compiles and test on Ubuntu
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        java: ["11"]
+      fail-fast: false
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Java ${{ matrix.java }}
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.java }}
+        java-package: jdk # (jre, jdk, or jdk+fx) - defaults to jdk
+        architecture: x64
+    - name: Build/Compile and run unit tests in repo using Maven step
+    # We do not have set settings.xml
+    # sonar flag are not included - requires adding secrets into repo security
+      run: mvn clean verify -U -fae -e -B -DskipITests=true -Dmaven.test.error.ignore=true -Dmaven.test.failure.ignore=true
```

Comment:
Seems to me that we are missing settings.xml required by JBoss Tools

Reply:
@jeffmaury Yes we are, it is written in comment, I am curious if it works out of a box now. It is quite unfortunate that I need review approval for each change I plan to do without even testing it... This is suppose to be just test if we can live with PR check performed via Github Actions. I guess that I will find repo where I can experiment and return back here once it works flawlessly.